airflow-core/src/airflow/ui/src/i18n/locales/zh_TW/dashboard.json

```
@@ -0,0 +1,39 @@
+{
+  "welcome": "歡迎",
+  "stats": {
+    "stats": "統計",
+    "failedDags": "失敗的 DAGs",
+    "runningDags": "執行中的 DAGs",
+    "activeDags": "活躍的 DAGs",
+    "queuedDags": "排隊中的 DAGs"
+  },
+  "health": {
+    "health": "健康狀態",
+    "metaDatabase": "元資料庫",
+    "scheduler": "排程器",
+    "triggerer": "觸發器",
+    "dagProcessor": "DAG 處理器"
+  },
+  "poolSlots": "資源池配額",
+  "managePools": "管理資源池",
+  "history": "歷史記錄",
+  "dagRuns": "DAG 執行",
+  "taskInstances": "任務實例",
+  "states": {
+    "queued": "排隊中",
```

```suggestion
    "queued": "排程中",
```

not sure whether it's better, but the original one sounds cute. I quite like it

---

I like 排隊 better since it better illustrates the ti is just waiting (to be run), not in the process of doing something.